ceci/stage.py

```diff
@@ -147,8 +156,24 @@ def open_output(self, tag, wrapper=False, **kwargs):
         else:
             return obj.file
 
+    @property
```

Comment:
Ah, looks like this is very deliberate then - fine!  Will keep reading ...

Reply:
yes :-)